python/dataproc_templates/hbase/README.md

```diff
@@ -5,7 +5,9 @@ Template for reading files from Hbase and writing to Google Cloud Storage. It su
 ## Requirements
```

Comment:
can we add docker as a requirement. Also for pushing images from local to container register I had to run gcloud auth configure-docker can we capture this as well?

Reply:
The required setup for connection in between local and cloud is a predefined step. Addressed it in the read me.